index.src.html

```diff
@@ -519,14 +520,18 @@ <h3 id="fetch-integration">Fetch Integration</h3>
   execute the following step:
 
   9.  If |response|'s <a>header list</a> contains a header named
-      <a>`Clear-Site-Data`</a>, then execute [[#clear-response]] on
-      |response|.
+      <a>`Clear-Site-Data`</a> and the request's <a>credentials mode</a> permits
+      modifying credentials, then execute [[#clear-response]] on |response|.
```

Comment:
@annevk SGTM. So I'll leave it here for now.

@mikewest Updated. Changed it to bulletpoints as it was difficult to parenthesize the boolean operators in that sentence. When we move this to the `fetch` spec, it seems that we can shorten it to "has a credentials flag set".